Projects.md

```diff
@@ -0,0 +1,102 @@
+|-----------------------------|-----------------------------------|
+| [python_moztelemetry][pymt] | Python APIs for Mozilla Telemetry
+| [moztelemetry][mt]          | Scala APIs for Mozilla Telemetry
+
+[pymt]: https://github.com/mozilla/python_moztelemetry
+[mt]: https://github.com/mozilla/moztelemetry
+
+## ETL code and Datasets
+
+| Name and repo               | Description                           |
+|-----------------------------|---------------------------------------|
+| [telemetry-batch-view][tbv] | Scala ETL code for derived datasets
+| [python_mozetl][pyetl]      | Python ETL code for derived datasets
+
+See also [firefox-data-docs][docs] for documentation on datasets.
+
+[tbv]: https://github.com/mozilla/telemetry-batch-view
+[pyetl]: https://github.com/mozilla/python_mozetl
+
+
+## Infrastructure
+
+| Name and repo                       | Description                             |
+|-------------------------------------|-----------------------------------------|
+| [mozilla-pipeline-schemas][schemas] | JSON Schemas for Mozilla Telemetry data
+| [hindsight][hs]                     | Realtime data processing
+| [lua_sandbox][lsb]                  | Generic sandbox for safe data analysis
+| [lua_sandbox_extensions][lsbx]      | Modules and packages that extend the Lua sandbox
+| [nginx_moz_ingest][nmi]             | Nginx module for Telemetry data ingestion
+| [Generic Ingestion][gi]             | Proposal for streamlined data ingestion of structured data
+| [puppet-config][puppet]             | Cloud services puppet config for deploying infrastructure
+
+[schemas]: https://github.com/mozilla-services/mozilla-pipeline-schemas
+[hs]: https://github.com/mozilla-services/hindsight
+[lsb]: https://github.com/mozilla-services/lua_sandbox
+[lsbx]: https://github.com/mozilla-services/lua_sandbox_extensions
+[nmi]: https://github.com/mozilla-services/nginx_moz_ingest
+[gi]: https://docs.google.com/document/d/1PqiF1rF2fCk_kQuGSwGwildDf4Crg9MJTY44E6N5DSk/edit?ts=5910c4cf#heading=h.74qlucdvwdg0
+[puppet]: https://github.com/mozilla-services/puppet-config/tree/master/pipeline
+
+### EMR Bootstrap scripts
+
+| Name and repo                     | Description                             |
+|-----------------------------------|-----------------------------------------|
+| [emr-bootstrap-spark][eb_spark]   | AWS bootstrap scripts for Spark.
+| [emr-bootstrap-presto][eb_presto] | AWS bootstrap scripts for Presto.
+| [emr-bootstrap-hbase][eb_hbase]   | AWS bootstrap scripts for HBase.
+
+[eb_spark]: https://github.com/mozilla/emr-bootstrap-spark
+[eb_presto]: https://github.com/mozilla/emr-bootstrap-presto
+[eb_hbase]: https://github.com/mozilla/emr-bootstrap-hbase
+
+## Data applications
+
+| Name and repo                   | Description                             |
+|---------------------------------|-----------------------------------------|
+| [telemetry.mozilla.org][tmo_gh] | Main entry point for viewing [aggregate Telemetry data][tmo]
```

Comment:
[Cerberus](https://github.com/mozilla/cerberus) and [Medusa](https://github.com/mozilla/medusa) should be included here as well.